docs/installing.rst

```diff
@@ -154,7 +154,7 @@ The following virtualization providers are suggested for the respective operatin
 
 For some operating systems, you might need to install additional dependencies before you install the Vagrant plugins.
 
-* Fedora 24
+* Fedora 22/23/24
```

Comment:
I think we should take this opportunity to clean up all F22 entries in the documentation as F22 is already end of life.